### ISSUE TITLE
ppa show update for the same version even after update

### ISSUE THREAD

**Issue by Muhammad-2021** [cboxdoerfer/fsearch#400]
**Describe the bug**
when i use the ppa for installing fsearch, and if a new version released it appear in the updates of my package manager so i update it, but when i open the package manager in another day i found an update for the same version.
for example my package manager need to update 0.2.2-0~202208221106~ubuntu20.04.1 to 0.2.2-0~202208231101~ubuntu20.04.1

**Expected behavior**
i should update to the new version only one time

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04

**Comment by cboxdoerfer** [comment]
Yes, it seems the PPA rebuilds for every commit in the release branch. I'll see if I can pin that to a certain commit or tag.

**Comment by Muhammad-2021** [comment]
ok, thanks
